README.md

```diff
@@ -48,7 +49,7 @@ Currently, we manually create the translation files necessary for Transifex to i
 
 To do this, run the following at the command line after following the steps below to build the documentation locally.
 
-1. Run the command in the /docs folder `sphinx-build -b gettext . docs_translations`
+1. Run the command in the `/docs` folder `sphinx-build -b gettext . docs_translations`
```

Comment:
We are not actually translating the docs as we couldn't get it to work, so I'd prefer if we removed this whole section (about doing translations) until we actually get it working properly.